gateway/src/apicast/policy/http_proxy/proxy.lua

```diff
@@ -0,0 +1,48 @@
+local policy = require('apicast.policy')
+local _M = policy.new('http_proxy')
```

Comment:
Can you please add the version? It's needed in order to make the feature of checking the order of policies work.
I needed to add the version in all the policies here: https://github.com/3scale/APIcast/pull/1088/files
In the future we might want to see if there's a better way to do this.